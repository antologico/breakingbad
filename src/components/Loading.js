import React from 'react'
import { FormattedMessage } from 'react-intl'
import { LoadingWrapper } from '../ui'
import loader from '../images/loader.svg'

export default () => <LoadingWrapper>
    <img src={loader} height='100px' />
    <div>
        <FormattedMessage id='loading' defaultMessage='...' />
    </div>
</LoadingWrapper>
