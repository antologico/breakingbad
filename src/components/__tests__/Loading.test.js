import React from 'react'
import { create } from '../../tests/utils'
import Loading from '../Loading'

jest.mock('../../images/loader.svg', () => 'loader')

jest.mock('react-router-dom', () => ({
  Link: ({children}) => <a id='link'>{children}</a>
}))

describe('Loading component', () => {
  test('renders properly', () => {
    const component = create(
        <Loading />
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})