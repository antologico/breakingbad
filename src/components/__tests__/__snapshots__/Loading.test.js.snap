// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders properly 1`] = `
<div
  className="sc-jSMfEi sc-gKXOVf cbmkWi hmSQRH"
>
  <img
    height="100px"
    src="loader"
  />
  <div>
    Cargando
  </div>
</div>
`;
